my-app/src/App.css

```diff
@@ -0,0 +1,246 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-image: url('./img/build.jpg');
+  background-size: cover;
+  background-position: right bottom;
+  min-height: 30vh;
+  max-height: 20vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+  font-family: Avenir;
+  font-weight: normal;
+  text-shadow: 3px 2px rgb(49, 41, 41);
+}
+
+.App-link {
+  color: #f8f8f8;
```

Comment:
It would be nice to have a theme file where you set all the colors you're going to use there.